app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -41,3 +42,67 @@ acceptance("Keyboard Shortcuts", function (needs) {
     assert.equal(currentURL(), "/t/keyboard-shortcuts-are-awesome/27331");
   });
 });
+
+acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
+  let resetNewCalled = 0;
+  let markReadCalled = 0;
+  needs.user();
+  needs.pretender((server, helper) => {
+    let topicList = DiscoveryFixtures["/latest.json"];
+
+    // get rid of some of the topics and the more_topics_url
+    // so we consider them allLoaded and show the footer with
+    // the bottom dismiss button
+    topicList.topic_list.topics.splice(20, 30);
+    topicList.topic_list.more_topics_url = null;
+
+    server.get("/unread.json", () => {
+      return helper.response(topicList);
+    });
+    server.get("/new.json", () => {
+      return helper.response(topicList);
+    });
+    server.put("/topics/reset-new", () => {
+      resetNewCalled += 1;
+      return helper.response({});
+    });
+    server.put("/topics/bulk", () => {
+      markReadCalled += 1;
+      return helper.response({});
+    });
+  });
+
+  test("dismiss unread from top and bottom button", async function (assert) {
+    await visit("/unread");
+    await triggerKeyEvent(document, "keypress", "x".charCodeAt(0));
+    await triggerKeyEvent(document, "keypress", "t".charCodeAt(0));
+    assert.ok(exists("#dismiss-read-confirm"));
+    assert.equal(
+      queryAll(".modal-body").text().trim(),
+      "Stop tracking these topics so they never show up as unread for me again"
```

Comment:
Instead of hardcoding the strings, can we use the translation helpers so that editing the translation in the future wouldn't break this test.